.travis.yml

```diff
@@ -21,9 +20,10 @@ services:
 # Env, before_script, and script for test stage
 env:
   matrix:
-    - "ANSIBLE_VER=2.8.20"
-    - "ANSIBLE_VER=2.9.20"
-    - "ANSIBLE_VER=2.10.7"
+    - "ANSIBLE_VER=2.8.20 ANSIBLE_PACKAGE=ansible"
+    - "ANSIBLE_VER=2.9.20 ANSIBLE_PACKAGE=ansible"
+    - "ANSIBLE_VER=2.10.7 ANSIBLE_PACKAGE=ansible"
```

Comment:
Any reason not to use `2.10.8` here?

Reply:
There is no `2.10.8` version of `ansible` -- that version only exists for `ansible-base`. The following is from a docker build of ansible, attempting to use 2.10.8.

```
#11 0.676 ERROR: No matching distribution found for ansible==2.10.8
#11 ERROR: executor failed running [/bin/sh -c pip install $ANSIBLE_PACKAGE==$ANSIBLE_VER]: exit code: 1
```